.devcontainer/rust/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM ubuntu:22.04@sha256:77906da86b60585ce12215807090eb327e7386c8fafb5402369e421f44eff17e
```

Comment:
why is this a different sha than the `cpp` version?

Reply:
Yes, it should be able to. But before the merge I will make sure they are in-sync.